plugin/src/integrationTest/kotlin/com/vanniktech/maven/publish/Subjects.kt

```diff
@@ -130,66 +131,67 @@ open class ArtifactSubject internal constructor(
       failWithoutActual(fact("expected not to exist", signedArtifact))
     }
   }
-}
-
-class SourcesJarSubject private constructor(
```

Comment:
This diff comes from moving the old implementation of `containsAllSourceFiles` to `containsFiles` in `ArtifactsSubject`